docs/_posts/examples/3400-01-27-3d-extrusion-floorplan.html

```diff
@@ -0,0 +1,60 @@
+--- 
+layout: example 
+category: example 
+title: Extrude polygons for 3D buildings.
+description: Create a 3D indoor map with the fill-extrude-height paint property.
+tags: 
+  - layers 
+---
+<div id='map'>
+</div>
+<script>
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v9',
+    center: [-87.61694, 41.86625],
+    zoom: 15.99,
+    pitch: 40,
+    bearing: 20
+});
+
+map.on('load', function() {
+
+    // Geojson Data source used in vector tiles, documented at
+    // https://gist.github.com/ryanbaumann/a7d970386ce59d11c16278b90dde094d
+    map.addSource("museumData", {
+        'type': 'geojson',
+        'data': '{{site.baseurl}}/assets/data/indoor-3d-map.geojson'
+    });
+
+    map.addLayer({
+        'id': 'room-extrude',
+        'type': 'fill',
+        'source': 'museumData',
+        'paint': {
+            // Color each polygon extrusion.
+            'fill-color': {
+                // Get the fill-color from the source 'color' property.
+                'property': 'color',
+                'type': 'identity'
+            },
+            // Extrude each polygon to a given height in meters.
+            'fill-extrude-height': {
+                // Get the fill-extrude-height from the source 'height' property.
+                'property': 'height',
+                'type': 'identity'
+            },
+            // Start extruding a polygon at a given height.  This allows for "floating" objects.
+            // For example, extruding from fill-extrude-base : 10 to fill-extrude-height: 20 will
+            // create a polygon floating above the map, starting at a height of 10 meters, extruded
+            // up to a max height of 20 meters.
```

Comment:
I have mixed feelings about this comment. One one hand, it is redundant to the style spec documentation. On the other hand, this may be the first time readers are seeing these properties used and the context may be helpful.